modules/javafx.media/src/main/native/gstreamer/projects/linux/avplugin/Makefile

```diff
@@ -33,7 +33,7 @@ CFLAGS = -fPIC                   \
          -DGLIB_VERSION_MAX_ALLOWED=GLIB_VERSION_2_48 \
          -ffunction-sections -fdata-sections
 
-ifneq ($(ARCH), aarch64)
+ifeq (,$(findstring $(ARCH), aarch64 loongarch64))
```

Comment:
This will work, but I wonder if we should revert the test: we now always assume that msse2 is supported, unless we are on an architecture which we know doesn't support it. It might be safer to only ask for msse2 support if we are really sure it is supported by the architecture (which currently means a check on x32 and x64)

Reply:
Done.